docs/services/vision/segmentation.md

```diff
@@ -200,6 +201,82 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+
+Use the segmenter to identify well separated objects above a flat plane.
+
+If you want to identify point clouds over the flat plane with your segmenter, [configure your frame system](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot, including your [camera](/components/camera/), within Viam's [frame system service](/services/frame-system/).
+After configuring your frame system, your camera will populate its own `Properties` with these spatial intrinsic parameters from the frame system.
+You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
+Then, set the segmenter's attribute `with_geometries: true`.
+The segmenter now returns point clouds within the `GeometryInFrame` object it captures.
+
+If you choose not to configure the frame system, you can still identify points over the flat plane with your segmenter.
```

Comment:
```suggestion
If you choose not to configure the frame system, you can still identify points over the flat plane with your segmenter.
```
@npentrel Tried to rephrase this section to make it more clear what they are doing, which is getting the representation in one form or another. The attribute is just a safety check to determine what kind of representation you're doing instead of checking whether or not you've configured the frame system behind the scenes and deciding what representation from there-- AFAIK it could be done either way but this just allows the user to get points or point clouds if they want to, making it their choice. But if they want point clouds they must do frame system config.